src/refresh/vkpt/vkpt.h

```diff
@@ -182,6 +182,7 @@ typedef struct QVK_s {
 	VkSwapchainKHR              swap_chain;
 	VkSurfaceFormatKHR          surf_format;
 	bool                        surf_is_hdr;
+	bool                        surf_write_srgb;
```

Comment:
Looks like this is unnecessary and just left over from the first PR.

Reply:
Duh, yes